cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
Note that even if code coverage is enabled (by the cmake variable `CODE_COVERAGE = on` ) that still does not mean the code coverage targets, .e.g, `unittest_coverage` will be built. (By default they will not be built.)